unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MinimapMetadata.cs

```diff
@@ -34,6 +34,18 @@ public class MinimapSceneInfo
         public string owner;
         public string description;
         public string previewImageUrl;
+
+        public override bool Equals(object obj)
+        {
+            if (obj is not MinimapSceneInfo) return false;
+            return Equals(obj as MinimapSceneInfo);
+        }
+
+        private bool Equals(MinimapSceneInfo other) =>
+            name.Equals(other.name) && owner.Equals(other.owner) && description.Equals(other.description);
```

Comment:
Only use this if `GetHashCode` is memoized
```suggestion
        private bool Equals(MinimapSceneInfo other)
        {
            return name.GetHashCode() == other.GetHashCode() && name.Equals(other.name) && owner.Equals(other.owner) && description.Equals(other.description);
        }
```